Demos/mpu6050/source/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <cstdint>
+
+#include "L1_Peripheral/lpc40xx/i2c.hpp"
+#include "L2_HAL/sensors/movement/accelerometer/mpu6050.hpp"
+#include "L2_HAL/sensors/movement/accelerometer/mma8452q.hpp"
+#include "utility/log.hpp"
+
+int main()
+{
+  sjsu::LogInfo("Mpu6050 Application Starting...");
+  sjsu::LogInfo(
+      "This example will access an MPU6050 accelerometer on the I2C bus. If "
+      "the accelerometer is accesable, then it will print out the accerometer "
+      "readings for each axis");
+
+  sjsu::lpc40xx::I2c i2c(sjsu::lpc40xx::I2c::Bus::kI2c2);
+  sjsu::Mpu6050 sensor(i2c, 2_SG);
+
+  sjsu::LogInfo("Initializing accelerometer peripherals...");
+  SJ2_RETURN_VALUE_ON_ERROR(sensor.Initialize(), 1);
+
+  sjsu::LogInfo("Enabling accelerometer...");
+  SJ2_RETURN_VALUE_ON_ERROR(sensor.Enable(), 1);
```

Comment:
```suggestion
  SJ2_RETURN_VALUE_ON_ERROR(sensor.Enable(), 2);
```
Set these to different numbers to indicate where the failure occured.